developer/src/kmc-kmn/src/kmw-compiler/keymanweb-key-codes.ts

```diff
@@ -268,4 +268,555 @@ export const
+"K_U",					// &H55
+"K_V",					// &H56
+"K_W",					// &H57
+"K_X",					// &H58
+"K_Y",					// &H59
+"K_Z",					// &H5A
+"K_?5B",				// &H5B
+"K_?5C",				// &H5C
+"K_?5D",				// &H5D
+"K_?5E",				// &H5E
+"K_?5F",				// &H5F
+"K_NP0",				// &H60
+"K_NP1",				// &H61
+"K_NP2",				// &H62
+"K_NP3",				// &H63
+"K_NP4",				// &H64
+"K_NP5",				// &H65
+"K_NP6",				// &H66
+"K_NP7",				// &H67
+"K_NP8",				// &H68
+"K_NP9",				// &H69
+"K_NPSTAR",			// &H6A
+"K_NPPLUS",			// &H6B
+"K_SEPARATOR",			// &H6C
+"K_NPMINUS",			// &H6D
+"K_NPDOT",				// &H6E
+"K_NPSLASH",			// &H6F
+"K_F1",				// &H70
+"K_F2",				// &H71
+"K_F3",				// &H72
+"K_F4",				// &H73
+"K_F5",				// &H74
+"K_F6",				// &H75
+"K_F7",				// &H76
+"K_F8",				// &H77
+"K_F9",				// &H78
+"K_F10",				// &H79
+"K_F11",				// &H7A
+"K_F12",				// &H7B
+"K_F13",				// &H7C
+"K_F14",				// &H7D
+"K_F15",				// &H7E
+"K_F16",				// &H7F
+"K_F17",				// &H80
+"K_F18",				// &H81
+"K_F19",				// &H82
+"K_F20",				// &H83
+"K_F21",				// &H84
+"K_F22",				// &H85
+"K_F23",				// &H86
+"K_F24",				// &H87
+
+"K_?88",				// &H88
+"K_?89",				// &H89
+"K_?8A",				// &H8A
+"K_?8B",				// &H8B
+"K_?8C",				// &H8C
+"K_?8D",				// &H8D
+"K_?8E",				// &H8E
+"K_?8F",				// &H8F
+
+"K_NUMLOCK",			// &H90
+"K_SCROLL",			// &H91
+
+"K_?92",				// &H92
+"K_?93",				// &H93
+"K_?94",				// &H94
+"K_?95",				// &H95
+"K_?96",				// &H96
+"K_?97",				// &H97
+"K_?98",				// &H98
+"K_?99",				// &H99
+"K_?9A",				// &H9A
+"K_?9B",				// &H9B
+"K_?9C",				// &H9C
+"K_?9D",				// &H9D
+"K_?9E",				// &H9E
+"K_?9F",				// &H9F
+"K_?A0",				// &HA0
+"K_?A1",				// &HA1
+"K_?A2",				// &HA2
+"K_?A3",				// &HA3
+"K_?A4",				// &HA4
+"K_?A5",				// &HA5
+"K_?A6",				// &HA6
+"K_?A7",				// &HA7
+"K_?A8",				// &HA8
+"K_?A9",				// &HA9
+"K_?AA",				// &HAA
+"K_?AB",				// &HAB
+"K_?AC",				// &HAC
+"K_?AD",				// &HAD
+"K_?AE",				// &HAE
+"K_?AF",				// &HAF
+"K_?B0",				// &HB0
+"K_?B1",				// &HB1
+"K_?B2",				// &HB2
+"K_?B3",				// &HB3
+"K_?B4",				// &HB4
+"K_?B5",				// &HB5
+"K_?B6",				// &HB6
+"K_?B7",				// &HB7
+"K_?B8",				// &HB8
+"K_?B9",				// &HB9
+
+"K_COLON",				// &HBA
+"K_EQUAL",				// &HBB
+"K_COMMA",				// &HBC
+"K_HYPHEN",			// &HBD
+"K_PERIOD",			// &HBE
+"K_SLASH",				// &HBF
+"K_BKQUOTE",			// &HC0
+
+"K_?C1",				// &HC1
+"K_?C2",				// &HC2
+"K_?C3",				// &HC3
+"K_?C4",				// &HC4
+"K_?C5",				// &HC5
+"K_?C6",				// &HC6
+"K_?C7",				// &HC7
+"K_?C8",				// &HC8
+"K_?C9",				// &HC9
+"K_?CA",				// &HCA
+"K_?CB",				// &HCB
+"K_?CC",				// &HCC
+"K_?CD",				// &HCD
+"K_?CE",				// &HCE
+"K_?CF",				// &HCF
+"K_?D0",				// &HD0
+"K_?D1",				// &HD1
+"K_?D2",				// &HD2
+"K_?D3",				// &HD3
+"K_?D4",				// &HD4
+"K_?D5",				// &HD5
+"K_?D6",				// &HD6
+"K_?D7",				// &HD7
+"K_?D8",				// &HD8
+"K_?D9",				// &HD9
+"K_?DA",				// &HDA
+
+"K_LBRKT",				// &HDB
+"K_BKSLASH",			// &HDC
+"K_RBRKT",				// &HDD
+"K_QUOTE",				// &HDE
+"K_oDF",				// &HDF
+"K_oE0",				// &HE0
```

Comment:
Who do we have `K_?..` for some keys and for others `K_o..`?

Reply:
This is quite historical; it maps to the Windows allocations. Those with `K_o` are designated as "OEM" keys in Windows, which probably means that some hardware keyboard driver has used them somewhere at some time. `K_?` are "reserved" and probably unused anywhere.